packages/opentelemetry-metrics/src/export/Batcher.ts

```diff
@@ -55,11 +51,15 @@ export class UngroupedBatcher extends Batcher {
   aggregatorFor(metricDescriptor: MetricDescriptor): Aggregator {
     switch (metricDescriptor.metricKind) {
       case MetricKind.COUNTER:
-        return new CounterSumAggregator();
-      case MetricKind.OBSERVER:
-        return new ObserverAggregator();
+      case MetricKind.UP_DOWN_COUNTER:
+      case MetricKind.SUM_OBSERVER:
+      case MetricKind.UP_DOWN_SUM_OBSERVER:
+        return new aggregators.SumAggregator();
+      case MetricKind.VALUE_RECORDER:
+      case MetricKind.VALUE_OBSERVER:
+        return new aggregators.LastValueAggregator();
       default:
-        return new ValueRecorderExactAggregator();
+        return new aggregators.MinMaxSumCountAggregator();
```

Comment:
Why is this the default aggregator? Does that come from spec?

Reply:
yes https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/api.md#interpretation
and the latest updates for aggregators are here too
https://github.com/open-telemetry/opentelemetry-specification/issues/636